packages/kusama/src/assetHubKusama.scheduler.test.ts

```diff
@@ -0,0 +1,5 @@
+import { kusama } from '@e2e-test/networks/chains'
```

Comment:
filename should be `assetHubKusama.scheduler.e2e.test.ts`

Reply:
True, thanks.